### DISCUSSION TITLE
Registering to all Geoman events

### DISCUSSION THREAD

**Discussion by cubecleveland** [geoman-io/leaflet-geoman#1091] [General]
Hello,
Is there a way to subscribe to all Geoman events in 1 handler.
Trying to implement a Redo fucntion on our map to Redo last edit and want to keep the state of previous edit....
thanks!

**Comment by Falke-Design** [comment]
With following code you can catch every event:
```
L.PM.Utils._fireEventOld = L.PM.Utils._fireEvent;
L.PM.Utils._fireEvent = function(layer, type, data, propagate = false) {
  L.PM.Utils._fireEventOld(layer, type, data, propagate);
  console.log(type);
}
```

**Reply by cubecleveland** [reply]
Awesome!
